src/main/frontend/db/query_dsl.cljs

```diff
@@ -141,6 +141,31 @@
      l)
     @vars))
 
+;; TODO: Convert -> fns to rules
+(defn ->property-query
+  ([k v]
+   (->property-query k v '?v))
+  ([k v sym]
+   [['?b :block/properties '?prop]
+    [(list 'missing? '$ '?b :block/name)]
+    [(list 'get '?prop (keyword k)) sym]
+    (list
+     'or
+     [(list '= sym v)]
+     [(list 'contains? sym v)]
+     ;; For integer pages that aren't strings
+     [(list 'contains? sym (str v))])]))
```

Comment:
I could've put logic to not add this if v is already a string but I want to have these queries be as dumb as possible. We need to clean up our inputs so we don't add complexity in rule creation